CHANGES.md

```diff
@@ -1,5 +1,40 @@
 # Changes
 
+## edge-21.2.2
+
+This edge release introduces support for multicluster TCP. It prepares support
+for multicluster opaque ports, which will follow-up in a future release.
+
+The `repair` command was added which will repopulate resources needed for
+upgrading from a `2.9.x` installation. There will be an error message during the
+upgrade process indicating that this command should be run so that users do not
+need to guess.
+
+Lastly, it contains a breaking change for Helm users upgrading with a customized
+`values.yaml`. If upgrading without this customization, there is no action to
+take.
+
+* **Breaking change**: Removed the `Global` field from the Linkerd Helm chart
+  now that it is unused because of the extension model; users doing upgrades
+  with Helm passing customized `values.yaml` will have to take this change into
+  account.
```

Comment:
Actually, helm installations are _always_ customized, because they need to include the issuer certs. So people will be forced to pass-in the certs again, no longer rooted at `global` (alongside any other customizations they had before, using the new structure).
Also tiny nit: `global` is all lower-cased.

Reply:
When I mentioned issuer certs above, I should have said instead `identityTrustAnchorsPEM` which is the mandatory config that was formerly under `global`.

Reply:
Okay I explained this up top.

Reply:
I'm thinking the verbiage could be something like this:

> **Breaking change for Helm**: The `values.yaml` entries for the linkerd2 Helm chart had several entries under the `global` scope, which now have been moved into the root scope. You need to pass again to `helm upgrade` any value that you have customized, using the new structure. At the very least, you'll need to provide the same `identityTrustAnchorsPEM` entry you previously used, which is always required and was moved into the root scope.